[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.NjdsjXAD9q"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:31 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:40 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:49 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(348): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(292): lmtttmpts_create_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_limit-...', false)\n#8 /internal/eval.php(20): activate_plugin('limit-attempts/...')\n#9 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by activate_plugin, do_action('activate_limit-attempts/limit-attempts.php'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_activate, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(361): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(292): lmtttmpts_create_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_limit-...', false)\n#8 /internal/eval.php(20): activate_plugin('limit-attempts/...')\n#9 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by activate_plugin, do_action('activate_limit-attempts/limit-attempts.php'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_activate, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_start` DATETIME AFTER `block_quantity`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `BLOCK_START`, datetime</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `block_start` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(429): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_start` DATETIME AFTER `block_quantity`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(348): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(432): lmtttmpts_create_table()\n#4 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-settings.php(727): do_action('init')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(361): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(432): lmtttmpts_create_table()\n#4 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-settings.php(727): do_action('init')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_by` TEXT AFTER `block_till`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `BLOCK_BY`, text</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `block_by` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(455): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_by` TEXT AFTER `block_till`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `email` TEXT AFTER `ip_int`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `EMAIL`, text</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `email` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(460): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `email` TEXT AFTER `ip_int`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `last_failed_attempt` TIMESTAMP AFTER `block_by`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `LAST_FAILED_ATTEMPT`, timestamp</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `last_failed_attempt` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(465): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `last_failed_attempt` TIMESTAMP AFTER `block_by`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_all_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_all_failed_attempts` ADD `email` TEXT AFTER `ip_int`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_all_failed_attempts, `EMAIL`, text</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_all_failed_attempts ADD `email` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_all_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(470): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_all_failed_attempts` ADD `email` TEXT AFTER `ip_int`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: BOOL.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_all_failed_attempts` ADD `block` BOOL DEFAULT FALSE AFTER `failed_attempts`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: BOOL.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(475): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_all_failed_attempts` ADD `block` BOOL DEFAULT FALSE AFTER `failed_attempts`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_lmtttmpts_denylist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_denylist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(513): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE wp_lmtttmpts_denylist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_lmtttmpts_allowlist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_allowlist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(513): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE wp_lmtttmpts_allowlist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"ADD\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:51 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_lmtttmpts_failed_attempts DROP PRIMARY KEY, ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: DROP INDEX &quot;wp_lmtttmpts_failed_attempts__,&quot; ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"ADD\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(506): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE wp_lmtttmpts_failed_attempts DROP PRIMARY KEY, ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"ADD\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:51 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_lmtttmpts_all_failed_attempts DROP PRIMARY KEY, ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: DROP INDEX &quot;wp_lmtttmpts_all_failed_attempts__,&quot; ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"ADD\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(506): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE wp_lmtttmpts_all_failed_attempts DROP PRIMARY KEY, ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(348): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(292): lmtttmpts_create_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_limit-...', false)\n#8 /internal/eval.php(20): activate_plugin('limit-attempts/...')\n#9 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by activate_plugin, do_action('activate_limit-attempts/limit-attempts.php'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_activate, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(361): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(292): lmtttmpts_create_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_limit-...', false)\n#8 /internal/eval.php(20): activate_plugin('limit-attempts/...')\n#9 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by activate_plugin, do_action('activate_limit-attempts/limit-attempts.php'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_activate, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_start` DATETIME AFTER `block_quantity`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `BLOCK_START`, datetime</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `block_start` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(429): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_start` DATETIME AFTER `block_quantity`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(348): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(432): lmtttmpts_create_table()\n#4 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-settings.php(727): do_action('init')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(361): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(432): lmtttmpts_create_table()\n#4 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-settings.php(727): do_action('init')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_by` TEXT AFTER `block_till`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `BLOCK_BY`, text</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `block_by` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(455): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_by` TEXT AFTER `block_till`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `email` TEXT AFTER `ip_int`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `EMAIL`, text</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `email` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(460): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `email` TEXT AFTER `ip_int`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `last_failed_attempt` TIMESTAMP AFTER `block_by`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `LAST_FAILED_ATTEMPT`, timestamp</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `last_failed_attempt` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(465): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `last_failed_attempt` TIMESTAMP AFTER `block_by`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_all_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_all_failed_attempts` ADD `email` TEXT AFTER `ip_int`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_all_failed_attempts, `EMAIL`, text</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_all_failed_attempts ADD `email` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_all_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(470): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_all_failed_attempts` ADD `email` TEXT AFTER `ip_int`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: BOOL.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_all_failed_attempts` ADD `block` BOOL DEFAULT FALSE AFTER `failed_attempts`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: BOOL.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(475): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_all_failed_attempts` ADD `block` BOOL DEFAULT FALSE AFTER `failed_attempts`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_lmtttmpts_denylist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_denylist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(513): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE wp_lmtttmpts_denylist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_lmtttmpts_allowlist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_allowlist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(513): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE wp_lmtttmpts_allowlist\n\t\t\t\t\t\t\tDROP `ip_from`,\n\t\t\t\t\t\t\tDROP `ip_to`,\n\t\t\t\t\t\t\tDROP `ip_from_int`,\n\t\t\t\t\t\t\tDROP `ip_to_int`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"ADD\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_lmtttmpts_failed_attempts DROP PRIMARY KEY, ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: DROP INDEX &quot;wp_lmtttmpts_failed_attempts__,&quot; ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"ADD\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(506): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE wp_lmtttmpts_failed_attempts DROP PRIMARY KEY, ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"ADD\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:01 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_lmtttmpts_all_failed_attempts DROP PRIMARY KEY, ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: DROP INDEX &quot;wp_lmtttmpts_all_failed_attempts__,&quot; ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"ADD\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(506): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE wp_lmtttmpts_all_failed_attempts DROP PRIMARY KEY, ADD `id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Uncaught TypeError: session_start(): Argument #1 ($options) must be of type array, string given in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:32 UTC] PHP Fatal error:  Uncaught TypeError: session_start(): Argument #1 ($options) must be of type array, string given in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): session_start('')\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-settings.php(727): do_action('init')\n#4 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#5 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#6 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#7 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Uncaught TypeError: session_start(): Argument #1 ($options) must be of type array, string given in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:43 UTC] PHP Fatal error:  Uncaught TypeError: session_start(): Argument #1 ($options) must be of type array, string given in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): session_start('')\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-settings.php(727): do_action('init')\n#4 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#5 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#6 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#7 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Uncaught TypeError: session_start(): Argument #1 ($options) must be of type array, string given in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:59:52 UTC] PHP Fatal error:  Uncaught TypeError: session_start(): Argument #1 ($options) must be of type array, string given in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): session_start('')\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-settings.php(727): do_action('init')\n#4 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#5 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#6 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#7 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Declaration of LBCB_Kuler_List_Table::prepare_items($criteria) must be compatible with WP_List_Table::prepare_items() in /wordpress/wp-content/plugins/lb-colorbox/include/lbcb-list-table.php on line 124",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 07:02:43 UTC] PHP Fatal error:  Declaration of LBCB_Kuler_List_Table::prepare_items($criteria) must be compatible with WP_List_Table::prepare_items() in /wordpress/wp-content/plugins/lb-colorbox/include/lbcb-list-table.php on line 124",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Declaration of LBCB_Kuler_List_Table::prepare_items($criteria) must be compatible with WP_List_Table::prepare_items() in /wordpress/wp-content/plugins/lb-colorbox/include/lbcb-list-table.php on line 124",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 07:02:51 UTC] PHP Fatal error:  Declaration of LBCB_Kuler_List_Table::prepare_items($criteria) must be compatible with WP_List_Table::prepare_items() in /wordpress/wp-content/plugins/lb-colorbox/include/lbcb-list-table.php on line 124",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Declaration of LBCB_Kuler_List_Table::prepare_items($criteria) must be compatible with WP_List_Table::prepare_items() in /wordpress/wp-content/plugins/lb-colorbox/include/lbcb-list-table.php on line 124",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 07:03:00 UTC] PHP Fatal error:  Declaration of LBCB_Kuler_List_Table::prepare_items($criteria) must be compatible with WP_List_Table::prepare_items() in /wordpress/wp-content/plugins/lb-colorbox/include/lbcb-list-table.php on line 124",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "link-connects",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.0Sz2DIg19j"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:31 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:40 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:49 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(348): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(292): lmtttmpts_create_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_limit-...', false)\n#8 /internal/eval.php(20): activate_plugin('limit-attempts/...')\n#9 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by activate_plugin, do_action('activate_limit-attempts/limit-attempts.php'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_activate, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(361): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(292): lmtttmpts_create_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_limit-...', false)\n#8 /internal/eval.php(20): activate_plugin('limit-attempts/...')\n#9 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by activate_plugin, do_action('activate_limit-attempts/limit-attempts.php'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_activate, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_start` DATETIME AFTER `block_quantity`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `BLOCK_START`, datetime</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `block_start` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(429): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_start` DATETIME AFTER `block_quantity`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(348): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(432): lmtttmpts_create_table()\n#4 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-settings.php(727): do_action('init')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "link-connects",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.PE2RpfXvwY"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:31 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:40 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:56:49 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>login-recaptcha</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(348): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(292): lmtttmpts_create_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_limit-...', false)\n#8 /internal/eval.php(20): activate_plugin('limit-attempts/...')\n#9 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by activate_plugin, do_action('activate_limit-attempts/limit-attempts.php'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_activate, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(361): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(292): lmtttmpts_create_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_limit-...', false)\n#8 /internal/eval.php(20): activate_plugin('limit-attempts/...')\n#9 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_all_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(4) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by activate_plugin, do_action('activate_limit-attempts/limit-attempts.php'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_activate, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_start` DATETIME AFTER `block_quantity`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_lmtttmpts_failed_attempts, `BLOCK_START`, datetime</li>\n<li>Executing: ALTER TABLE wp_lmtttmpts_failed_attempts ADD `block_start` text  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_lmtttmpts_failed_attempts.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(429): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-settings.php(727): do_action('init')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_lmtttmpts_failed_attempts` ADD `block_start` DATETIME AFTER `block_quantity`; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  },
  {
    "message": "Unexpected token in MySQL query: block.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "link-connects",
    "details": "[25-Apr-2025 06:58:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unexpected token in MySQL query: block.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE IF...')\n#2 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(348): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(432): lmtttmpts_create_table()\n#4 /wordpress/wp-content/plugins/limit-attempts/limit-attempts.php(171): register_lmtttmpts_settings()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): lmtttmpts_plugin_init('')\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-settings.php(727): do_action('init')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query CREATE TABLE IF NOT EXISTS `wp_lmtttmpts_failed_attempts` (\n            `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,\n            `ip` CHAR(31) NOT NULL,\n            `ip_int` BIGINT,\n            `email` VARCHAR( 255 ),\n            `failed_attempts` INT(3) NOT NULL DEFAULT '0',\n            `block` BOOL DEFAULT FALSE,\n            `block_quantity` INT(3) NOT NULL DEFAULT '0',\n            `block_start` DATETIME,\n            `block_till` DATETIME,\n            `block_by` VARCHAR( 255 ),\n            `last_failed_attempt` TIMESTAMP,\n            PRIMARY KEY (`id`)\n            ) DEFAULT CHARSET=utf8 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, lmtttmpts_plugin_init, register_lmtttmpts_settings, lmtttmpts_create_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/l/link-connects/error.json"
  }
]
